Firmware/Marlin_main.cpp

```diff
@@ -11705,8 +11721,12 @@ void restore_print_from_ram_and_continue(float e_move)
 	{
 		setTargetHotendSafe(saved_extruder_temperature, saved_active_extruder);
 		heating_status = HeatingStatus::EXTRUDER_HEATING;
+		// The global fanSpeedBckp is intentionally not used here.
+		uint8_t fanSpeedBackup = fanSpeed;
+		fanSpeed = 0; // Turn off print fan while extruder is heating
 		wait_for_heater(_millis(), saved_active_extruder);
 		heating_status = HeatingStatus::EXTRUDER_HEATING_COMPLETE;
+		fanSpeed = fanSpeedBackup; // Restore previous speed setting for print fan
```

Comment:
I don't see any principal problem here.
I just needs to be tested if it doesn't induce big temperature dip in the moment the print fan is turned on.
If it would be found this is the problem, more complicated alternative approach would be to resume fan e.g. in the moment extruder heater temperature is closer than 10°C to target temperature.
In such case there would be enough time for integral part of regulation term to adapt for running fan.

Reply:
This part I'm not worried about, this is perfectly safe.

Just that M106/M109 S/R could set the fan speed as a result of this PR.
Technically, the same effect can be achieved by reordering the instructions already at the g-code level.